Source/DiabloUI/selok.cpp

```diff
@@ -34,7 +34,7 @@ void selok_Free()
 	vecSelOkDialog.clear();
 }
 
-void selok_Select(int value)
+void selok_Select(int /*value*/)

```

Comment:
I'm not sure I see why that the point here is, I image it's to remove the warning about an unused argument. Commented code is usually bad though (especially using C-style), and I'd rather disable that warning of debatable usefulness...

Reply:
This is how clang-tidy automatically deals with that issue yeah. I could manually delete the name instead...

Reply:
Ah that's a bit annoying because you can't see the argument name anymore. Why not disable that warning?